$OpenBSD$

Disable another Kad assertion probably resulting from bad data from
external sources
(upstream git commit 7d6176bf769edf8f3076889a57361a3abb2fae57)

--- src/kademlia/routing/RoutingZone.cpp.orig	Sat Oct  8 19:16:30 2011
+++ src/kademlia/routing/RoutingZone.cpp	Wed May  9 14:59:58 2012
@@ -812,7 +812,7 @@ void CRoutingZone::OnSmallTimer()
 			CKademlia::GetUDPListener()->SendMyDetails(KADEMLIA2_HELLO_REQ, c->GetIPAddress(), c->GetUDPPort(), c->GetVersion(), 0, NULL, false);
 			wxASSERT(c->GetUDPKey() == CKadUDPKey(0));
 		} else {
-			wxFAIL;
+			//wxFAIL;
 		}
 	}
 }
